_includes/footer.html

```diff
@@ -3,6 +3,28 @@
 		<a class="logo" href="{{ site.baseurl }}/">
 			<img src="{{ site.baseurl }}/img/logo_white.png">
 		</a>
+		<center>
+	<div class = "flex-container">
+		<ul style="display:flex; list-style-type:none">
+		{% for onepage in site.pages %} 
+		{% if onepage.layout == "default" and onepage.title != "Home" and onepage.title != "Projects"%}
```

Comment:
Looks like you went through a lot of trouble to make the hrefs dynamic-- you can actually just use href="{{ onepage.url }}" to do what you were trying to do with this if statement mess! :)

Reply:
Forgot to mention-- it probably has to be href="{{ site.baseurl }}/{{ onepage.url }}". Baseurl basically refers to the relative path from the root directory of the server.